### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by Tim-Jordison**
- updates to readme to make more consistent
- readme now links to drop-in gift card docs
- really minor update to checkout.html

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,29 @@
...
-- `LOGO`: the logo for the store.
 - `SHOPPER_ID`: the shopper that will be interacting with the checkout page as well as the default shopper used for account management.
 - `ORDER_TOTAL`: the total amount the shopper will be charged at checkout.
 
-If the shopper with that shopperId doesn't exist or doesn't have a USD account yet, go to the account management page to create and fund the account.
+If the shopper with that `shopperId` doesn't exist or doesn't have a USD account yet, go to the account management page to create and fund the account.
 
-## Running
+### Step 2: Drop-in Gift Card Setup
+You'll need your Drop-in Gift Card [template](https://www.lightrail.com/app/#/cards/dropin) within your Lightrail account.
```
Comment (tjukes) [inline README.md]: "You'll need to **set up**..."

File: node/README.md [hunk 2]
```diff
@@ -1,21 +1,12 @@
-# Lightrail Stripe Sample Node Web App
```
Comment (tjukes) [inline node/README.md]: Maybe worth it to keep a main title with the environment name in it?

File: README.md [hunk 3]
```diff
@@ -1,25 +1,30 @@
 # Lightrail Stripe Sample Web App
+This project demonstrates how to use Lightrail's Drop-in Gift Card components. See our [documentation](https://github.com/Giftbit/Lightrail-API-Docs/blob/master/docs/quickstart/drop-in-gift-cards.md#drop-in-gift-cards) for a quick overview.
```
Comment (pushplay) [inline README.md]: An overview of what?

File: README.md [hunk 4]
```diff
@@ -1,25 +1,30 @@
 # Lightrail Stripe Sample Web App
+This project demonstrates how to use Lightrail's Drop-in Gift Card components. See our [documentation](https://github.com/Giftbit/Lightrail-API-Docs/blob/master/docs/quickstart/drop-in-gift-cards.md#drop-in-gift-cards) for a quick overview.  
 
-This project demonstrates how to use Lightrail and Stripe to create an account, add credit to an account, and use the account credit on Stripe checkout.  It uses Mustache templates and language-specific libraries for Lightrail and Lightrail-Stripe
-
-The concepts used by this demo are covered in the use-case document [https://github.com/Giftbit/Lightrail-API-Docs/blob/master/use-cases/account-credits.md](https://github.com/Giftbit/Lightrail-API-Docs/blob/master/use-cases/account-credits.md).  It's worth scanning that document before continuing.
+It uses Mustache templates and language-specific libraries for Lightrail and Lightrail-Stripe
```
Comment (pushplay) [inline README.md]: This paragraph seems to be unfocused.  I think what it needs to do is introduce what technologies are must use and what choices are incidental for the demo.

> This demo is implemented in 2 environments: NodeJS and PHP.  (Ruby and Java examples are coming soon.)  Each environment uses language-specific libraries for Lightrail and Lightrail-Stripe that we strongly recommend.  Common display code is rendered using Mustache templates, but any similar technology will do.

File: node/README.md (hunk 2 above)
Comment (pushplay) [reply node/README.md]: Not having a titular level 1 header is going to look weird in any app that generates a table of contents for a markdown file (like docs app).

**Pull request merged**